### DISCUSSION TITLE
不應滿足現狀，蔡明介：台灣半導體業仍要結構升級

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#166] [General]
https://technews.tw/2021/12/20/taiwan-semiconductor-industry-commented-by-m-k-tsai/
 不應滿足現狀，蔡明介：台灣半導體業仍要結構升級

**Comment by pocper1** [comment]
前50年已經奠定有台積電，若再結構升級，能再奠定未來50年，半導體的龍頭國

**Comment by qeeeee39** [comment]
前一陣子全球缺鐵缺鋼加上疫情，因此晶圓大缺貨，在這個時候台積電直接成為世界超大晶圓供應商。
